### ISSUE TITLE
Text change on Access settings form

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#597]
1. Please change the text at the top of the page to an H2: "Access settings"
2. "Release date", "Individual file visibility", and "Download access" should be bold Paragraph.
